.circleci/config.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       S3_DB_BACKUPS_BUCKET: 'db-backups.parabol.co'
       SERVER_ID: '0'
     working_directory: ~/action
+    resource_class: medium+
```

Comment:
CPU and memory usage with the default `medium` executor was hitting 100%:

![Screen Shot 2022-02-01 at 11 44 22 AM](https://user-images.githubusercontent.com/630449/152031010-13341702-e6e2-49be-a7ad-7356621e9d10.png)

This will eventually cause flaky/failed builds. So, to avoid this, I bumped up to the [medium+ resource class](https://circleci.com/docs/2.0/configuration-reference/#docker-executor). With this resource runner, we have a bit more breathing room:

![Screen Shot 2022-02-01 at 11 46 00 AM](https://user-images.githubusercontent.com/630449/152031292-039fbe29-b44b-4136-81bb-3efab61d4539.png)

Pricing wise, this will increase cost from 10 credits/min to 15 credits/min ([docs](https://circleci.com/product/features/resource-classes/#docker)).

With your current free plan with CircleCI, you get 30,000 credits per month. Reviewing [past usage](https://app.circleci.com/settings/plan/github/ParabolInc/usage) it looks like you haven't used more than 14k in the last few months, so I believe this is a safe trade-off to make.

Alternatively, we can stick with `medium` for the time being and bump if we start running into transient failures.

Reply:
oh this is awesome!